webpack.config.js

```diff
@@ -270,7 +270,7 @@ module.exports = {
 		new DependencyExtractionWebpackPlugin( { injectPolyfill: true } ),
 	].filter( Boolean ),
 	watchOptions: {
-		ignored: '!packages/*/!(src)/**/*',
+		ignored: '!packages/*/build-module/**/*',
```

Comment:
Actually, this is going to watch everything expect the build-module folder which is the opposite of what we want it seems :P (the key is `ignored`)

Reply:
It's a double negation, we tell ignore everything that isn't in the `build-module` 🙃 

The API for `watchOptions` is very simple and doesn't provide the allowed list.

Reply:
oh didn't see that `!` there

Reply:
We will need RegExp to make it work with the PostCSS upgrade, it's even more fun:

```JS
ignored: /^(?!.*\/packages\/.*\/build-module\/.*)$/,
```